src/guides/libraries/c-interop.md

```diff
@@ -253,6 +253,58 @@ To reduce this burden,
 you can use the [`package:ffigen`][ffigen] binding generator
 to automatically create FFI wrappers from C header files.
 
+## Building and bundling native assets
+
+{{site.alert.note}}
+  The native assets feature is **experimental**,
+  and [in active development](https://github.com/dart-lang/sdk/issues/50565).
+{{site.alert.end}}
+
+The Native Assets feature aims to resolve a number of issues associated with the
+distribution of Dart packages that depend on native code. It does so by
+providing uniform hooks for integrating with various build systems involved
+in building Flutter and standalone Dart applications.
+
+With native assets, you can now include a `build.dart` script as part of your
```

Comment:
The `flutter run` / `flutter build` as well as `dart` and `dart build` tools will now build & bundle any native code. Those tools will look at all package dependencies and obtain binary artifacts (via invoking the package's `build.dart` script - see <links>) relevant for the platform one is building for.